### PR TITLE
[WIP] Update <new projects> from skeletor

### PR DESCRIPTION
**Pull request by ckcollab**
# @ mention of reviewers
@gibsonbailey 

# A brief description of the purpose of the changes contained in this PR.
With this hopefully we can at least _receive_ updates from skeletor. I may add the "push to skeletor" behavior in another PR or in this same one if I get a wild hair.

Can see #31 for more deets!

# Hand testing
Usage:
```bash
$ make update_from_skeletor

# (you'll need a SKELETOR_VERSION file, but you can also test it 
# prompts you to make one!)
```

- [ ] start project from this and putz with skeletor?
- [ ] could maybe run this against something we started off skeletor a while ago?

# Checklist
- [ ] Code review by me 
- [ ] New code covered by automated tests
- [ ] I'm proud of my work
- [ ] Code review by reviewer
- [ ] Hand tested by reviewer
- [ ] Ready to merge

### CODE REVIEW

File: bin/update_from_skeletor.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+SKELETOR_VERSION=$(cat $SKELETOR_LOCATION)
+
+# Try to see if there's anything new...
+PATCH=$(curl -s "https://github.com/ckc-org/skeletor/compare/${SKELETOR_VERSION}...master.patch")
+
+if [ ! -z "$PATCH" ]; then
+    SKELETOR_VERSION=$(get_skeletor_master_sha)
+    echo "Updating ./SKELETOR_VERSION to ${SKELETOR_VERSION}"
+    echo $SKELETOR_VERSION > $SKELETOR_LOCATION
```
Comment (gibsonbailey) [inline bin/update_from_skeletor.sh:21]: Should we update this only if the patch succeeds?

**Comment by ckcollab** [comment]
Use different style of creating a patch:

https://unix.stackexchange.com/questions/162131/is-this-a-good-way-to-create-a-patch

**Comment by ckcollab** [comment]
using `patch -p1` does kinda work but man is this funky... one thing to consider is with a whole new install script, _maybe_ we can semi-reasonably fork skeletor and leave skeletor around as a remote we can do diffs/patches against..!

could name the remote [snake-mountain](https://evil.fandom.com/wiki/Snake_Mountain) for fun ;)

**Comment by ckcollab** [comment]
Probably instead of doing this, we'll have separate modules (like django-ckc or django-chit-chat) for carrying large pieces of functionality forward...

**Pull request closed without merging**